src/main/java/com/EighthLight/app/App.java

```diff
@@ -0,0 +1,14 @@
+package com.EighthLight.app;
+
+public class App {
+    public static void main (String[] args) {
+        int size = 3;
+        Player playerOne = new Player("X");
+        Player playerTwo = new Player("O");
+        Ui ui = new Ui(System.in);
```

Comment:
It might make more sense to pass in something with an type that you control here. You aren't guaranteed that every UI will have an input stream that you can inject. As such it's better to pass in something that adheres to an interface that you control.